packages/indexer-common/src/indexer-management/models/indexing-rule.ts

```diff
@@ -30,6 +30,24 @@ export interface IndexingRuleAttributes {
   requireSupported: boolean
 }
 
+export const IndexingRuleAttributeDictionary = [
```

Comment:
I don't expect the indexing rule models to change, but in order to ease management and make the code more "DRY" can we avoid creating a new constant here and instant derive the array of keys from the `IndexingRuleAttributes` interface or the `IndexingRule` class?

Reply:
I totally agree! 'Instant derive' is what I was missing. Do you think we could use module `ts-transformer-keys`?

Reply:
I think this should be doable without any additional libraries.  

Perhaps we can use the rawAttributes property of the Sequelize model?
`Object.keys(IndexingRule.rawAttributes)`?

or use the keys from an instance of the class?
`Object.keys(new IndexingRule())`?